druid/src/contexts.rs

```diff
@@ -349,7 +352,13 @@ impl<'a> EventCtx<'a> {
     /// request with the event.
     pub fn request_timer(&mut self, deadline: Instant) -> TimerToken {
         self.base_state.request_timer = true;
-        self.window.request_timer(deadline)
+        let timer_token = self.window.request_timer(deadline);
+        self.base_state.add_timer(timer_token);
+        timer_token
+    }
+
+    pub fn remove_timer(&mut self, timer_token: &TimerToken) {
```

Comment:
After widgets are done processing timer events, should we all ask them to remove map entry?
This is not used anywhere in examples. 
I think framework needs to remove it rather than forcing widgets to remove it themselves.

Reply:
After merging up, Im clearing the map of BaseState. So remove is not needed when event is handled.